embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/capture/metadata/RnBundleIdTrackerImpl.kt

```diff
@@ -164,9 +164,9 @@ public class RnBundleIdTrackerImpl(
             val bundleHashed = md.digest(bundle)
             val sb = StringBuilder()
             for (b in bundleHashed) {
-                sb.append(String.format(Locale.getDefault(), "%02x", b.toInt() and 0xff))
+                sb.append(String.format(Locale.ENGLISH, "%02x", b.toInt() and 0xff))
```

Comment:
This isn't a warning but when formatting something that isn't language based, we want consistency. Default can cause issues when a locale formats a string in an unexpected way